Library/Formula/rhino.rb

```diff
@@ -2,9 +2,9 @@
 
 class Rhino < Formula
   homepage 'https://www.mozilla.org/rhino/'
-  url 'https://github.com/mozilla/rhino/releases/download/Rhino1_7R5_RELEASE/rhino1_7R5.zip'
-  sha1 '39ada4cca8c7f651a68eda3f307ff9b68808f0ce'
-  version '1.7R5'
+  url 'https://github.com/mozilla/rhino/releases/download/Rhino1_7_6_RELEASE/rhino1.7.6.zip'
+  sha1 '7d86ccb422bc82569b334bad68cee385c3a4540c'
+  version '1.7.6'
```

Comment:
You can remove this line since Homebrew is able to extract the version from the URL.